AIMSICD/src/main/java/com/SecUpwN/AIMSICD/utils/Status.java

```diff
@@ -27,11 +25,11 @@
 
     public enum Type {
```

Comment:
everything that is *greyed out* (commented out) in our code doesn't exist in the compiled app